bokehjs/src/lib/core/util/bbox.ts

```diff
@@ -362,19 +394,19 @@ export class BBox implements Rect, Equatable {
 
   resolve(symbol: string): XY | number {
     switch (symbol) {
```

Comment:
If at all, then it would be `this[symbol] ?? {x: NaN, y: NaN}`. Leave `||` for boolean logic and use `??` for `null` coalescing. However, this will not work, because `symbol` can be anything and we want to filter out things that exist on `this`, but are not valid symbols, e.g. `this["constructor"]`. Although, one could image `"constructor"` symbol that resolves to something that is not the actual constructor. This also wouldn't work on type-level, because one can't index `this` with a `string`. You would need a `keyof this`, or something like that, but then you wouldn't be able to use this method in any context it's used in. A pattern like this would work for sealed objects of limited scope, like enumerations, with which we use it quite often.